target/linux/bcm27xx/patches-6.1/950-0081-smsx95xx-fix-crimes-against-truesize.patch

```diff
@@ -25,21 +25,19 @@ Signed-off-by: Steve Glendinning <steve.glendinning@smsc.com>
  static int __must_check smsc95xx_read_reg(struct usbnet *dev, u32 index,
  					  u32 *data)
  {
-@@ -1870,7 +1874,8 @@ static int smsc95xx_rx_fixup(struct usbn
+@@ -1875,6 +1879,8 @@ static int smsc95xx_rx_fixup(struct usbn
  				if (dev->net->features & NETIF_F_RXCSUM)
  					smsc95xx_rx_csum_offload(skb);
  				skb_trim(skb, skb->len - 4); /* remove fcs */
--				skb->truesize = size + sizeof(struct sk_buff);
 +				if (truesize_mode)
 +					skb->truesize = size + sizeof(struct sk_buff);
```

Comment:
I just had a short look and this looks strange, that we add back the truesize although this commit also complains that truesize override should not be done at this stage:
https://github.com/torvalds/linux/commit/d50729f1d60bca822ef6d9c1a5fb28d486bd7593

Reply:
@PolynomialDivision It makes sense, but it is temporarily retained due to no device testing. May I ask if your opinion no longer requires `truesize`？

Reply:
Maybe @Noltari can say something about it? I don't understand why we even have this patch. It does not look like target is loading the driver with truesize_mode enabled? I would be in favor for removing this patch.

Reply:
@Noltari Please give me your opinion.

Reply:
I think we can remove this patch from OpenWrt.
I do not have the hardware or know anything special about it.
The patch in OpenWrt deactivates setting `skb->truesize` by default, `truesize_mode` is `false` by default. 
The upstream kernel change removes setting it completely. 
https://github.com/gregkh/linux/commit/517e64bcc9634bb46f8fc6421e272b8da3d7d953

I do not see any code in OpenWrt setting the `truesize_mode` module parameter.

Reply:
@zxlhhyccc Please remove the patch. I would remove it before doing the bump in a seperate commit.

Reply:
@PolynomialDivision I will remove the patch and resubmit it later.